src/main/java/life/qbic/data/processing/evaluation/EvaluationRequest.java

```diff
@@ -209,18 +227,11 @@ private void moveBackToOrigin(File taskDir, Provenance provenance, String reason
     }
   }
 
-  private void moveToTargetDir(File taskDir) {
+  private void copyToTargetDir(File taskDir) throws IOException {
     LOG.info(
-        "Moving %s to target directory %s".formatted(taskDir.getAbsolutePath(),
+        "Copying %s to target directory %s".formatted(taskDir.getAbsolutePath(),
             assignedTargetDirectory));
```

Comment:
Hi @sven1103 
Shouldn't the assigned target directory be an absolute path as well?

Reply:
Since it is a path and was configured absolute, the absolute path will be shown here